ssl/statem/statem_srvr.c

```diff
@@ -587,13 +618,26 @@ static WRITE_TRAN ossl_statem_server13_write_transition(SSL_CONNECTION *s)
          * Following an initial handshake we send the number of tickets we have
          * been configured for.
          */
-        if (!SSL_IS_FIRST_HANDSHAKE(s) && s->ext.extra_tickets_expected > 0) {
-            return WRITE_TRAN_CONTINUE;
-        } else if (s->hit || s->num_tickets <= s->sent_tickets) {
+        if (!(!SSL_IS_FIRST_HANDSHAKE(s) && s->ext.extra_tickets_expected > 0)
```

Comment:
```suggestion
        if ((SSL_IS_FIRST_HANDSHAKE(s) || s->ext.extra_tickets_expected <= 0)
```

To help make this boolean logic a little clearer